R/getGeneticInteractome.R

```diff
@@ -0,0 +1,62 @@
+# getGeneticInteractome.R
+
+#' Return BioGrid genetic interaction tags of physical interactors of the system
+#'
+#' @param mySys The dataframe of PPI between system's components
+#' @param key  a unique string key
+#' @param criterion A string, either "stringent" or "relaxed", specifying whether only GGI between physical interactors should be selected.
+#' @return The dataframe of system components and either their GGI between physical interactors should be selected (iff \code{criterion} == "stringent")
+#' or all GGI of physical interactors (iff \code{relaxed} == "stringent")
+#' @import xlsx
+#' @import readxl
+#' @importFrom dplyr filter
+#' @import biomaRt
+#' @import ggplot2
+#' @import biogridr
+#' @import utils
+#' @importFrom stats complete.cases
+#' @export
+getGeneticInteractome <- function(mySys, criterion, key) {
+  myGenes <-
+    toString(unique(c(
+      as.character(mySys$a),
+      as.character(mySys$b)
+    )))
+
+  myGenes <- gsub(", ", "|", myGenes)
+  humInt <-
```

Comment:
I'm not a fan of `%>%` at all since it is much harder to debug functions when you can't inspect intermediate values.  But this is of course technically valid code. In situations such as these I usually write something like:
```
x <- f1()
x <- f2(x)
x <- f3(x)
```

Reply:
Fixed.